k8s-risk-assessment-job/values.yaml

```diff
@@ -32,3 +32,4 @@ clusterName: ""
 clusterID: 0
 label: ""
 secretName: ""
+airgapped: false
```

Comment:
we should keep this true by default ... Better to not pull changes from the GH dynamically.